CalendarApp/BookingRoom/Controllers/UpdateController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using BookingRoom.Models;
+using BookingRoom.Models.GoogleCalendar;
+using Google.Apis.Requests;
+using Google.Apis.Calendar.v3.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using BookingRoom.Models.GoogleConnection;
+using BookingRoom.Models.GoogleEvent;
+using log4net;
+
+namespace BookingRoom.Controllers
+{
+    public class UpdateController : EventController
+    {
+        public UpdateController(ICalendarConnection connection,ILog logger)
+            : base(connection,logger)
+        {
+
+        }
+        [HttpPut]
+        public HttpResponseMessage UpDateEvent(CalendarEvent eventForUpdate,string eventID)
+        {
+            Event updateEvent = ToEventConverter.ToEvent(eventForUpdate);        
+            try
+            {
+                _connection.GoogleCalendar.Events.Update(updateEvent, eventForUpdate.CalendarID, eventID).Execute();
+            }
+            catch(Exception e)
+            {
+                log.Error("Exception -\n" + e);
```

Comment:
The log using present config:<param name ="ConversionPattern" value ="%newline *************%date%newline %logger - %message%newline*************%date%newline"/>
   will look like:
 *************2015-09-28 02:23:28,549
 ExceptionLogger - Exception -
The service calendar has thrown an exception: Google.GoogleApiException: Google.Apis.Requests.RequestError
Not Found [404]
Errors [
	Message[Not Found] Location[ - ] Reason[notFound] Domain[global]
]

   at Google.Apis.Requests.ClientServiceRequest`1.Execute() in c:\code\github\google-api-dotnet-client\Tools\Google.Apis.Release\bin\Release\1.9.2\default\Src\GoogleApis\Apis\Requests\ClientServiceRequest.cs:line 93
   at BookingRoom.Controllers.UpdateController.UpDateEvent(CalendarEvent eventForUpdate, String eventID) in C:\Users\Dmytro\Desktop\ProjectFolder\CalenderApp\CalendarApp\BookingRoom\Controllers\UpdateController.cs:line 30
*************2015-09-28 02:23:28,549